data/Language/JSON.yaml

```diff
@@ -0,0 +1,15 @@
+identifier: JSON
+wikidata: Q2063
+full_name: JavaScript Object Notation
+extensions: 
+  - json
+line_continuation: ''
+delimiters:
+  # From coala JSON language definition
+  # JSON don't have any comments
+  # String delimiter:
+  - double_quote
+  # Block delimiter:
+  - curly_braces
+  - square_braces
+keywords: []
```

Comment:
You can remove this line as there are no keywords.

Reply:
I was unsure about what to do with that field, I'll remove it now